v2/bigtable-changestreams-to-hbase/README.md

```diff
@@ -8,19 +8,53 @@ The [BigtableToHBase](src/main/java/com/google/cloud/teleport/v2/templates/Bigta
 * Bigtable table with change streams enabled
 * Hbase instance that is accessible from Dataflow
 
-#### Beware
+#### Caveat - provision Hbase resources to handle Dataflow traffic
 
 * Dataflow when strongly provisioned can translate Bigtable input QPS directly to Hbase, the user should configure Hbase so that Hbase can handle that QPS without crashing.
+
 * Bigtable change streams can handle very large single mutations. Hbase write buffers should be adequately provisioned so as not to cause exceptions when writing in large mutations.
 
+#### Caveat - data divergence can occur in extreme edge case
+
+Data divergence can occur in the following scenario:
+* Dataflow assigns a Put to worker A
```

Comment:
You should add something here  on the lines of (All mutations for a row will go to same worker).

Reply:
Added